main.py

```diff
@@ -569,18 +569,9 @@ def do_whisper(audio_file, model:str, beam_size:int = beam_size, task:str = "tra
     return language, results, infer_time_milliseconds, translation, infer_speedup, audio_duration
 
 
-# Handy function for converting numbers to the individual word
```

Comment:
This function exists because some numbers need to be spoken individually. For example, it's common for addresses to be pronounced as "seven-zero-one" instead of the current num2words approach of "seven hundred and one". It is unused but at a minimum we would eventually like do so some NLP processing and/or API parameter for integration/use with WIS and the various applications on the forthcoming Willow Application Server.

Reply:
Ok, would you like me to put it back in for this PR?

Reply:
Yes please - wisng is VERY new and probably at least a week from merge. It needs A LOT of testing and as I've noted before I will likely make a wisng endpoint available so the broader community can easily test. There's no possible way we've nailed what wisng is today :).